src/context/walletContext/walletContext.tsx

```diff
@@ -8,7 +8,13 @@ import { useQuery, useQueryClient } from '@tanstack/react-query'
 import { PEANUT_WALLET_CHAIN, PEANUT_WALLET_TOKEN } from '@/constants'
 import { Chain, erc20Abi, getAddress, parseUnits } from 'viem'
 import { useAuth } from '../authContext'
-import { backgroundColorFromAddress, areEvmAddressesEqual, fetchWalletBalances } from '@/utils'
+import {
+    backgroundColorFromAddress,
+    areEvmAddressesEqual,
+    fetchWalletBalances,
+    getFromLocalStorage,
+    saveToLocalStorage,
```

Comment:
good on using existing functions!

Thought: we should clean up general.utils.ts, we save tons of shit to localStorage in random ways. Need to clean this up - probably same time we redo our tx indexing